crates/core/tedge_mapper/src/aws/mapper.rs

```diff
@@ -59,3 +73,30 @@ fn get_topic_filter(tedge_config: &TEdgeConfig) -> TopicFilter {
     }
     topics
 }
+
+fn built_in_bridge_rules(remote_client_id: &str) -> Result<BridgeConfig, anyhow::Error> {
```

Comment:
Could this `BridgeConfig` also used to generate the [mosquitto bridge config](https://github.com/thin-edge/thin-edge.io/blob/main/crates/core/tedge/src/bridge/aws.rs#L19)? This would avoid to repeat ourselves.